README.md

```diff
@@ -69,7 +66,7 @@ var options = {
   elk: {
     // All options are available at http://www.eclipse.org/elk/reference.html
     // 'org.eclipse.elk.' can be dropped from the Identifier
-    // Or look at demo.html for an example.
+    // Or look at demo-demo.js for an example.
```

Comment:
Typo?

Reply:
No – the `demo.html` file is now just a shell that opens various demo files; the original `demo.html`'s script is `demo-demo.js`.

Reply:
OK